dough-data/src/test/java/io/github/bakedlibs/dough/collections/DummyData.java

```diff
@@ -0,0 +1,49 @@
+package io.github.bakedlibs.dough.collections;
+
+import javax.annotation.Nonnull;
+
+class DummyData implements Cloneable {
+
+    private final int value;
+
+    public DummyData(int value) {
```

Comment:
This class is `package-private`, therefore the `public` modifier here is misleading and not usable to the degree of its specifications.

```suggestion
    DummyData(int value) {
```